### PR TITLE
Add models for operator dashboard

### PR DESCRIPTION
**Pull request by bmtcril**
Also removes a number of user_pii fields that are currently unused and cause ClickHouse to fail to build them when run at scale.

### CODE REVIEW

**Comment by Ian2012** [comment]
@bmtcril how does this works on local and dev?

Where you able to query the data locally?

File: models/users/dim_user_pii.sql [hunk 1]
```diff
@@ -1,17 +1 @@
-select
-    external_user_id,
```
Comment (bmtcril) [inline models/users/dim_user_pii.sql:2]: The extra fields in user_pii were running CH out of memory trying to create the dictionary on the 300M row instance. They are almost always null right now, and I think if we want to start using that data we need a separate "extended" model to join against later.

File: models/instance/fact_instance_activity.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{{
+    config(
+        materialized="materialized_view",
+        schema=env_var("ASPECTS_XAPI_DATABASE", "xapi"),
+        engine=get_engine("SummingMergeTree()"),
```
Comment (bmtcril) [inline models/instance/fact_instance_activity.sql:5]: I think this is our first use of a SummingMergeTree. I'm still trying to confirm this is all working as expected.

**Review by Ian2012 — APPROVED** [review]

File: models/instance/fact_instance_activity.sql (hunk 2 above)
Comment (SoryRawyer) [reply models/instance/fact_instance_activity.sql:5]: Were you able to confirm that this does what we want? It looks correct to me but I'm trying to test this against the `AggregatingMergeTree` table engine.

Comment (bmtcril) [reply models/instance/fact_instance_activity.sql:5]: There's something off with it when there is new data coming in, I'm still debugging it

**Review by SoryRawyer — APPROVED** [review]
I think this looks good! I tried testing the use of `AggregatingMergeTree` along with `uniqCombinedState` and it seemed to work as I'd expect.

**Pull request merged**